templates/ts/src/api.ts

```diff
@@ -0,0 +1,60 @@
+import { Player } from "./types/Player";
+import { GameMap } from "./types/GameMap";
+import { GameState } from "./types/GameState";
+
+const LOCAL_URL = "http://localhost:8080";
+const PROD_URL = "https://bots-of-black-friday.azurewebsites.net";
+
+const getUrl = (): string => {
+  if (process.env.NODE_ENV === "development") {
+    return LOCAL_URL;
+  }
+  return PROD_URL;
+};
+
+const headers = {
+  "content-type": "application/json;charset=UTF-8",
+};
+
+interface RegisterResponse {
```

Comment:
Jäin miettimään olisiko kuitenkin selkeämpi, jos tämä olisi types.ts-tiedostossa?